src/ResourceManager/Maps/Maps.Test/TestController.cs

```diff
@@ -182,9 +187,14 @@ private GalleryClient GetGalleryClient()
             return TestBase.GetServiceClient<GalleryClient>(this.csmTestFactory);
         }
 
-        private Client GetLocationBasedServicesManagementClient(RestTestFramework.MockContext context)
+        private MapsManagementClient GetMapsManagementClient(RestTestFramework.MockContext context)
+        {
+            return context.GetServiceClient<MapsManagementClient>(RestTestFramework.TestEnvironmentFactory.GetTestEnvironment());
+        }
+
+        private LegacyResourceManagementClient GetLegacyResourceManagementClient(RestTestFramework.MockContext context)
```

Comment:
What changed to mean that you need this? Did you change the tests?

Reply:
No changes were introduced.
If I recall, it worked ~2 months ago, but broke recently after pulling the latest code from the preview branch. This includes unmodified LocationBasedServices cmdlet tests, that yields the same error like shown below:
```
Test Name:	Microsoft.Azure.Commands.Maps.Test.ScenarioTests.MapsAccountTests.TestRemoveAccount
Test Outcome:	Failed
Result Message:	System.Management.Automation.ActionPreferenceStopException : The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: TestManagementClientHelper class wasn't initialized with the ResourceManagementClient client.
```